gmgc_mapper_plotter/GMGC_Mapper_plotter.ipynb

```diff
@@ -0,0 +1,222 @@
+      "text/plain": [
+       "<Figure size 720x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "hist_2= df[1].value_counts()\n",
+    "k_2 = len(df[1])\n",
+    "graf_3 = hist_2.plot(kind='barh', figsize=(10, 10), color='powderblue', zorder=5, width=0.5)\n",
+    "graf_3.tick_params(axis=\"both\", which=\"both\", bottom=\"off\", top=\"off\", labelbottom=\"on\", left=\"off\", right=\"off\", labelleft=\"on\")\n",
+    "vals_3 = graf_2.get_xticks()\n",
+    "for tick in vals_3:\n",
+    "      graf_3.axvline(x=tick, linestyle='dashed', alpha=0.8, color='lightgrey', zorder=1)\n",
+    "graf_3.set_xlabel(\"Counts\", labelpad=10, weight='bold', size=12)\n",
+    "graf_3.set_ylabel(\"Number of hits\", labelpad=10, weight='bold', size=12)\n",
+    "graf_3.set_title(\"GMGC mapper\", weight='bold', size=14)\n",
+    "for i, v in enumerate(hist_2):\n",
+    "    graf_3.text(v + 2 , i - 0.2, str(v)+'('+str(round(v*100/k_2 , 1))+'%)', color='teal', fontweight='bold',size=8)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = ['<10 hits', '10 - 25 hits', '25 - 50 hits', '>50 hits' ]\n",
+    "\n",
+    "k , j , l , m = 0 , 0 , 0 , 0\n",
```

Comment:
I believe this sequence of lines can be replaced by a single call to `pandas.cut` https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.cut.html